torch/_export/serde/schema.py

```diff
@@ -152,6 +159,9 @@ class Argument(_Union):
     as_graph: GraphArgument
     as_optional_tensors: List[OptionalTensorArgument]
 
+    def __repr__(self):
+        return super().__repr__()
+
```

Comment:
Why we need this? won't Python look up automatically for __repr__?

Reply:
dataclass generates a repr function, which overrides parent class' implementation. 
I guess I can also fix this by setting (repr=False)...